github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -299,6 +299,202 @@ periodics:
         resources:
           requests:
             memory: "34Gi"
+- name: periodic-kubevirt-e2e-k8s-1.20-sig-network
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cron: "0 1,9,17 * * *"
```

Comment:
This is the exact same time that the 1.19-sig-network periodic uses, could you pls shift it a bit so that we trigger the periodics separately and don't have the risk to DDoS ourselves? Same for the rest, I think they match the 1.19 periodics trigger times.

Reply:
Done